src/ExecutionTraceContext.ts

```diff
@@ -0,0 +1,25 @@
+import { TraceContext } from "./public/Interfaces";
+import { AzureFunctionsRpcMessages as rpc } from '../azure-functions-language-worker-protobuf/src/rpc';
+
+export function CreateTraceContext (requestTraceContext: rpc.IRpcTraceContext | null | undefined)
+{
+    return new ExecutionTraceContext(requestTraceContext);
+}
+
+class ExecutionTraceContext implements TraceContext
+{
+  Traceparent: string | null | undefined;
+  Tracestate: string | null | undefined;
+  Attributes: {
+    [k: string]: string
+  } | null | undefined;
+
+  constructor(requestTraceContext: rpc.IRpcTraceContext | null | undefined) {
+    if(requestTraceContext!==null && requestTraceContext!==undefined)
```

Comment:
`if (requestTraceContext)` is good too. Means "is not falsy", or is not:
- false
- 0 (zero)
- “” (empty string)
- null
- undefined
- NaN (Not A Number)

https://codeburst.io/javascript-null-vs-undefined-20f955215a2